benefit/data/sheets/CaseField/CaseField-sandl-nonprod.json

```diff
@@ -3,5 +3,7 @@
   {"LiveFrom": "08/03/2022", "CaseTypeID": "Benefit", "ID": "activeHearingVersionNumber", "Label": "Active Hearing Version Number", "FieldType": "Number", "SecurityClassification": "PUBLIC"},
   {"LiveFrom": "01/06/2022", "CaseTypeID": "Benefit", "ID": "overrideFields", "Label": "Overrides Listing Values", "FieldType": "overrideFields", "SecurityClassification": "PUBLIC"},
   {"LiveFrom": "01/06/2022", "CaseTypeID": "Benefit", "ID": "defaultOverrideFields", "Label": "Default Listing Values", "FieldType": "overrideFields", "SecurityClassification": "PUBLIC"},
-  {"LiveFrom": "06/07/2022", "CaseTypeID": "Benefit", "ID": "amendReasons", "Label": "Amend Reason", "FieldType": "MultiSelectList", "FieldTypeParameter": "FL_amendReason", "SecurityClassification": "PUBLIC"}
+  {"LiveFrom": "06/07/2022", "CaseTypeID": "Benefit", "ID": "amendReasons", "Label": "Amend Reason", "FieldType": "MultiSelectList", "FieldTypeParameter": "FL_amendReason", "SecurityClassification": "PUBLIC"},
+  {"LiveFrom": "04/07/2022", "CaseTypeID": "Benefit", "ID": "didPoAttend", "Label": "Did PO Attend?", "FieldType": "YesOrNo", "SecurityClassification": "PUBLIC"}
```

Comment:
didPoAttend is still here

Reply:
You want that one gone?, I thought it was the other